pkg/services/store/object/object.proto

```diff
@@ -325,4 +369,13 @@ service ObjectStore {
 // Ideally an additional search endpoint with more flexibility to limit what you actually care about
 //  https://github.com/grafana/grafana-plugin-sdk-go/blob/main/proto/backend.proto#L129
 //  rpc SearchEX(ObjectSearchRequest) returns (DataResponse);
+
```

Comment:
Will want to remove the AdminWrite from the main service and only use the explicit admin service, but that will require us to do additional wire+grpc registration, so I'll punt on that for now :)